test/Integration/SqlOutputBindingIntegrationTests.cs

```diff
@@ -355,7 +355,7 @@ public void AddProductCaseSensitiveTest(SupportedLanguages lang)
             this.StartFunctionHost(nameof(AddProductParams), lang);
 
             // Change database collation to case sensitive
-            this.ExecuteNonQuery($"ALTER DATABASE {this.DatabaseName} COLLATE Latin1_General_CS_AS");
+            this.ExecuteNonQuery($"ALTER DATABASE {this.DatabaseName} SET Single_User WITH ROLLBACK IMMEDIATE; ALTER DATABASE {this.DatabaseName} COLLATE Latin1_General_CS_AS; ALTER DATABASE {this.DatabaseName} SET Multi_User;");
```

Comment:
Modified this because I was getting the following error when running this test locally:
`The database could not be exclusively locked to perform the operation`